source/particles/particle_handler.cc

```diff
@@ -1087,11 +1072,6 @@ namespace Particles
         // not locally owned cells have to be resorted or transferred.
```

Comment:
Please remove this outdated comment. The comment should now be:
```
Do not sort particles that are not locally owned, because they will be sorted by the process that owns them.
```